airbyte-integrations/connectors/source-zoom/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   name: Zoom
   registries:
     cloud:
-      enabled: true
+      enabled: false
```

Comment:
I believe we need to update our documentation, but we shouldn't release it without conducting thorough testing.